autoload/jedi.vim

```diff
@@ -93,7 +93,7 @@ function! s:init_python()
 
         " Add a warning in case the auto-detected version is not available,
         " usually because of a missing neovim module in a VIRTUAL_ENV.
-        if has('nvim')
+        if has('nvim') && (!exists('g:python3_host_prog') && !exists('g:python_host_prog'))
```

Comment:
@achiku 
Line 90's right side of the `or` / `||` is used for Neovim.

The logic is: if it is Vim, or (for Neovim) `has('python')` works.

Feel free to fix / improve the flow in this area, but as far as I can see it should work as-is.

What is your use case?
You are using the host prog globals (which are correct), and get a warning?